src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/prison/UserService.kt

```diff
@@ -52,6 +52,8 @@ class UserService(
     return prisonUserDetails
   }
 
+  fun findUserByUserName(username: String) = prisonUserApiService.findUserByUsername(username)
```

Comment:
naming convention for findByUsername usually uses lowercase n (not sure why we use userName in other places, possibly just personal preference)

Reply:
Its done thanks.